backstop.config.js

```diff
@@ -1,56 +0,0 @@
-// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
-const glob = require('glob');
-
-function findAllPages() {
-  return glob.sync('**/*.page.tsx', { cwd: 'pages' }).map(file => file.replace(/\.page\.tsx$/, ''));
-}
-
-const skippedTests = [
-  'tabs/responsive-permutations', // Currently has flaky behavior (AWSUI-17810)
-  'pie-chart/permutations', // Currently has flaky behavior (AWSUI-29588)
```

Comment:
There were issues specific to this setup only. Our main screenshot infra does not have the same issues, so the related tickets will be resolved